### DISCUSSION TITLE
I2c with last firmware

### DISCUSSION THREAD

**Discussion by amo74** [arendst/Tasmota#21110] [Q&A]
Hi. 
I use the last firmware version of tasmota-sensors with D1 Mini and SHT31X sensor. 
![17123591107404772968724203779427](https://github.com/arendst/Tasmota/assets/64838514/82551efd-d803-4a39-ae0a-2655156ddd0c)
![17123591455361059846608536029170](https://github.com/arendst/Tasmota/assets/64838514/dae0d8d0-62a6-40d9-af85-332f742a276f)

![17123588964278031355332176250381](https://github.com/arendst/Tasmota/assets/64838514/891ab657-721d-44ea-ad53-06a0edd6de16)

But this is the main page:

![17123589685817733885676886954673](https://github.com/arendst/Tasmota/assets/64838514/8399d341-a508-4fb4-8772-d8965b9eb480)

In the console page I2CScan command does not works:

![17123590526526494354734121574866](https://github.com/arendst/Tasmota/assets/64838514/8fe38496-87eb-49a1-b629-ce7804d99d21)

### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
you have swapped SDA and SCL

**Reply by sfromis** [reply]
That's correct, but the first error is configuring a template without activating it. When you use a template, you need to enable it too, like by selecting module 0 (which is the template) instead of the default "Sonoff Basic".

**Reply by amo74** [reply]
Thank you to all. Now it works!
